ui/src/containers/billingplans.list/index.tsx

```diff
@@ -26,7 +26,7 @@ export default function PlanList() {
         const {
           // @ts-ignore
           data: { plans },
-        } = await client?.frontierServiceListPlans();
+        } = await client?.frontierServiceListPlans() ?? {};
```

Comment:
this will be more readable and we are using this convention everywhere
``` 
   const resp = await client?.frontierServiceListPlans();
   const plans = resp?.data?.plans
```